audit/projects/k8s-artifacts-prod/services/pubsub/subscriptions.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "ackDeadlineSeconds": 600,
+    "expirationPolicy": {},
+    "messageRetentionDuration": "604800s",
+    "name": "projects/k8s-artifacts-prod/subscriptions/cip-auditor-invoker",
```

Comment:
Expected, we're exporting pubsub subscriptions as a result of https://github.com/kubernetes/k8s.io/pull/2133

And this is setup via https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/ensure-env-cip-auditor.sh